### PR TITLE
move src/examples to examples

### PR DESCRIPTION
**Pull request by aconchillo**
I believe this will capture users' attention better. People just click at `examples`, plus I believe it is more common to have them in the root directory.

### CODE REVIEW

**Comment by aconchillo** [comment]
Oh... I think this is more tricky (since the examples depends on the examples package). I'll mark it as a draft.

**Comment by aconchillo** [comment]
@Moishe @chadbailey59 With these changes we don't make `examples` part of the `dailyai` distribution any more. Today, if you do `pip install dailyai` it will also install the examples. I believe the examples should just be in the repo and keep the distribution small.

To do that, since the `examples` fodler is also a python package (because of examples/support/runner.py) there are 2 options:

- User needs to specify `PYTHONPATH`. For example: `PYTHONPATH=. python/examples/foundational/02....`
- Each example file adds the repos root folder to the python search path. Which is what I've done here.

WDYT?

File: examples/foundational/01-say-one-thing.py [hunk 1]
```diff
@@ -8,7 +8,18 @@
...
+# autopep8: off
+# Make sure we load 'examples' package.
+import sys
+project_root = os.path.abspath(
+    os.path.join(
+        os.path.dirname(__file__),
+        '..',
+        '..',))
+sys.path.append(project_root)
+
```
[the remark this answers is not in the document]
Comment (aconchillo) [reply examples/foundational/01-say-one-thing.py:20]: Just added a comment about this. The other option is for the user to set `PYTHONPATH`.

[the remark this answers is not in the document]
Comment (aconchillo) [reply examples/foundational/01-say-one-thing.py:20]: Agree. I actually prefer the `PYTHONPATH` approach, but for some reason I went with the other.

[the remark this answers is not in the document]
Comment (aconchillo) [reply examples/foundational/01-say-one-thing.py:20]: We could add a line to the README. Something like:

```
To run the example you need to set PYTHONPATH.

export  PYTHONPATH=`pwd`
....
....
```

**Comment by aconchillo** [comment]
I've updated the PR with the `PYTHONPATH` approach.

**Comment by Moishe** [comment]
I don't love setting PYTHONPATH either, because it's gonna require extra setup steps to run the examples inside VSCode (or whatever other IDE).

What if we just put a copy of the runner.py file in `foundational` and `starter-apps` (and removed the `support` directory entirely)? In this case I think a little duplicated code in favor of ease of running the examples might be better.

(I tried using relative imports with no luck, maybe because it would also require a PYTHONPATH, just a different one)

**Comment by aconchillo** [comment]
> I don't love setting PYTHONPATH either, because it's gonna require extra setup steps to run the examples inside VSCode (or whatever other IDE).
> 
> What if we just put a copy of the runner.py file in `foundational` and `starter-apps` (and removed the `support` directory entirely)? In this case I think a little duplicated code in favor of ease of running the examples might be better.
> 

That works for me! I was also thinking of adding `runner.py` into the `dailyai` package inside a `util/daily` subpackage or something. But I like this idea better for now.

> (I tried using relative imports with no luck, maybe because it would also require a PYTHONPATH, just a different one)

Yes, I tried that as well without luck.

**Comment by aconchillo** [comment]
> > I don't love setting PYTHONPATH either, because it's gonna require extra setup steps to run the examples inside VSCode (or whatever other IDE).
> > What if we just put a copy of the runner.py file in `foundational` and `starter-apps` (and removed the `support` directory entirely)? In this case I think a little duplicated code in favor of ease of running the examples might be better.
> 
> That works for me! I was also thinking of adding `runner.py` into the `dailyai` package inside a `util/daily` subpackage or something. But I like this idea better for now.
> 

Actually, this has the same problem as the relative import as you would do:

```
from .runner import configure
```

But since we don't have a package....

**Comment by aconchillo** [comment]
Alright, I have moved `auth.py` and `runner.py` into `dailyai.examples` packages. This of course works well. Now you have to do:

```
from dailyai.examples.runner import configure
```

**Comment by Moishe** [comment]
I copied runner.py into the examples directory and changed the import to:

`from runner import configure`

and then, from the examples directory, 

```
(env) ➜  examples git:(move-src-example-to-examples) ✗ python foundational/01-say-one-thing.py -u https://moishe.daily.co/sample-testing
Using cache found in /Users/moishe/.cache/torch/hub/snakers4_silero-vad_master
20 2024-03-18 19:45:44,831 🎬 Starting frame consumer thread
```

that seems to work fine without adding any example-specific code to the package. Does that work for you?

**Comment by aconchillo** [comment]
> I copied runner.py into the examples directory and changed the import to:
> 
> `from runner import configure`
> 
> and then, from the examples directory,
> 
> ```
> (env) ➜  examples git:(move-src-example-to-examples) ✗ python foundational/01-say-one-thing.py -u https://moishe.daily.co/sample-testing
> Using cache found in /Users/moishe/.cache/torch/hub/snakers4_silero-vad_master
> 20 2024-03-18 19:45:44,831 🎬 Starting frame consumer thread
> ```
> 
> that seems to work fine without adding any example-specific code to the package. Does that work for you?

Duh! Of course it works. PR updated.

**Comment by aconchillo** [comment]
I kept `load_dotenv()` in each script though instead of inside `runner.py`.

**Review by Moishe — APPROVED** [review]

**Pull request merged**